src/useHasOverflow.ts

```diff
@@ -1,5 +1,10 @@
 import { createRef, useLayoutEffect, useState } from 'react'
 
+const hasElementOverflown = (elem: HTMLElement) => {
```

Comment:
`hasElementOverflow`? Or you need `n`? Or `isElementOverflown`?

Reply:
Renamed